core/src/main/java/me/leoko/advancedban/utils/Command.java

```diff
@@ -557,7 +558,19 @@ public void execute(Object player, String[] args) {
 
 
     public TabCompleter getTabCompleter() {
+        MethodInterface mi = Universal.get().getMethods();
+        if (!mi.getBoolean(mi.getConfig(), "Use Tab Completion", true)) {
+            return new TabCompleter() {
```

Comment:
We already have an implementation of empty tab-complete:
```suggestion
            return new NullTabCompleter();
```

Reply:
ah perfect haven't noticed it.